src/gui/tray/Window.qml

```diff
@@ -590,11 +584,11 @@ Window {
                     Text {
                         id: activityTextInfo
                         text: (type === "Sync") ? displayPath
-                            : (type === "File") ? subject
-                            : (type === "Notification") ? message
-                            : ""
+                                                : (type === "File") ? subject

```

Comment:
I wouldn't change the indentation there, I find it more readable the way it was before (felt more like pattern matching in a functional language).